var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -0,0 +1,690 @@
+
+            # Please specify which C++ compiler should be used as the host
+            # C++ compiler
+            env.set('HOST_CXX_COMPILER', spack_cxx)
+
+            # Please specify which C compiler should be used as the host
+            # C compiler
+            env.set('HOST_C_COMPILER', spack_cc)
+
+            # Do you wish to build TensorFlow with ComputeCPP support?
+            if '+computecpp' in spec:
+                env.set('TF_NEED_COMPUTECPP', '1')
+
+                # Please specify the location where ComputeCpp is installed
+                env.set('COMPUTECPP_TOOLKIT_PATH', spec['computecpp'].prefix)
+            else:
+                env.set('TF_NEED_COMPUTECPP', '0')
+
+                # Please specify the location of the triSYCL include directory
+                env.set('TRISYCL_INCLUDE_DIR', spec['trisycl'].prefix.include)
+        else:
+            env.set('TF_NEED_OPENCL_SYCL', '0')
+            env.set('TF_NEED_OPENCL', '0')
+
+        # Do you wish to build TensorFlow with ROCm support?
+        if '+rocm' in spec:
+            env.set('TF_NEED_ROCM', '1')
+        else:
+            env.set('TF_NEED_ROCM', '0')
+
+        # Do you wish to build TensorFlow with CUDA support?
+        if '+cuda' in spec:
+            env.set('TF_NEED_CUDA', '1')
+
+            # Do you want to use clang as CUDA compiler?
+            env.set('TF_CUDA_CLANG', '0')
+
+            # Please specify which gcc nvcc should use as the host compiler
+            env.set('GCC_HOST_COMPILER_PATH', spack_cc)
+
+            cuda_paths = [
+                spec['cuda'].prefix,
+                spec['cudnn'].prefix,
+            ]
+
+            # Do you wish to build TensorFlow with TensorRT support?
+            if '+tensorrt' in spec:
+                env.set('TF_NEED_TENSORRT', '1')
+
+                cuda_paths.append(spec['tensorrt'].prefix)
+
+                # Please specify the TensorRT version you want to use
+                env.set('TF_TENSORRT_VERSION',
+                        spec['tensorrt'].version.up_to(1))
+
+                # Please specify the location where TensorRT is installed
+                env.set('TENSORRT_INSTALL_PATH', spec['tensorrt'].prefix)
+            else:
+                env.set('TF_NEED_TENSORRT', '0')
+                env.unset('TF_TENSORRT_VERSION')
+
+            # Please specify the CUDA SDK version you want to use
+            env.set('TF_CUDA_VERSION', spec['cuda'].version.up_to(2))
+
+            # Please specify the cuDNN version you want to use
+            env.set('TF_CUDNN_VERSION', spec['cudnn'].version.up_to(1))
+
+            if '+nccl' in spec:
+                cuda_paths.append(spec['nccl'].prefix)
+
+                # Please specify the locally installed NCCL version to use
+                env.set('TF_NCCL_VERSION', spec['nccl'].version.up_to(1))
+
+                # Please specify the location where NCCL is installed
+                env.set('NCCL_INSTALL_PATH', spec['nccl'].prefix)
```

Comment:
@Sinan81 Should this also be `prefix.lib`? Should we set `NCCL_LIB_PATH` as well or does that not do anything?

Reply:
If I remember correctly, as of v1.14: `nccl_install_path=spec['nccl'].prefix` works. Don't remember using `nccl_lib_path` before.